blocks/footer/footer.js

```diff
@@ -13,7 +56,21 @@ export default async function decorate(block) {
   const resp = await fetch(`${footerPath}.plain.html`);
   const html = await resp.text();
   const footer = document.createElement('div');
+
   footer.innerHTML = html;
   await decorateIcons(footer);
   block.append(footer);
+  addCSSToLinkHeadings();
+
+  // code for building mobile footer
+  const mobileMedia = window.matchMedia('(max-width: 667px)');
```

Comment:
is it possible to avoid this? If not, not the end of the world, but generally having to rebuild a block as the window size changes is discourages